audience_manager/api.rst

```diff
@@ -3,173 +3,215 @@
 
-List audiences
-``````````````
-Gets list of user audiences.
+Get list of audiences user belongs to
+`````````````````````````````````````
+Gets list of :term:`audience` IDs :term:`user` belongs to.
 
 Code::
 
-    ppms.am.api('getAudiences', onFulfilled, onRejected);
+    ppms.am.api("getAudiences", onFulfilled, onRejected);
 
 .. function:: onFulfilled(audience_list)
 
     Function executed on success.
 
-    :param Array<string> audience_list: Array of audience IDs user belongs to. Example::
+    :param Array<string> audience_list: Array of :term:`audience` IDs :term:`user` belongs to.
+
+        Example::
 
-        ['e8c6e873-955c-4771-9fd5-92c94577e9d9', '756e5920-422f-4d13-b73a-917f696ca288']
+            ["e8c6e873-955c-4771-9fd5-92c94577e9d9", "756e5920-422f-4d13-b73a-917f696ca288"]
 
-.. function:: onRejected(error)
+.. function:: onRejected(error_code)
 
     Function executed on error.
 
-    :param string error: Error code. Example::
+    :param string error_code: Error code.
 
-        'server_error'
+        Example::
 
-Check audience
-``````````````
-Checks if user is part of specific audience.
+            "server_error"
+
+Check user membership in the audience
+`````````````````````````````````````
+Checks if :term:`user` belongs to specific :term:`audience`.
 
 Code::
 
-    ppms.am.api('checkAudience', audience_id, onFulfilled, onRejected);
+    ppms.am.api("checkAudience", audience_id, onFulfilled, onRejected);
 
 .. data:: audience_id
 
-    ID of checked audience. Example::
+    ID of checked :term:`audience`.
+
+    Example::
 
-        '52073260-5861-4a56-be5e-6628794722ee'
+        "52073260-5861-4a56-be5e-6628794722ee"
 
 .. function:: onFulfilled(in_audience)
 
     Function executed on success.
 
-    :param boolean in_audience: True when user is part of audience, false otherwise.
+    :param boolean in_audience: *True* when :term:`user` is part of the :term:`audience`, *false* otherwise.
 
-.. function:: onRejected(error)
+        Example::
+
+            true
+
+.. function:: onRejected(error_code)
 
     Function executed on error.
 
-    :param string error: Error code. Example::
+    :param string error_code: Error code.
+
+        Example::
 
-        'server_error'
+            "server_error"
 
-Get attributes
-``````````````
-Gets user attributes.
+Get user attributes
+```````````````````
+Gets :term:`user` profile :term:`attributes<attribute>`. :term:`User` has to be identified by :term:`analytics ID`.
 
-.. warning::
+.. note::
+    In order to secure the :term:`PII` data, no :term:`attribute` is returned by default. You need to put each
+    :term:`attribute` you want to access on :term:`attribute whitelist` before it'll be returned by this command. In
+    order to do that, go to `Audience Manager` > `Attributes` tab and `enable` :term:`attribute` for the public API
+    access. It's your responsibility to use this tool to make sure no :term:`user` :term:`PII` data will be available
```

Comment:
> _It's your responsibility to use this tool to make sure_
It sounds a bit odd. Propositions:
1. It's your responsibility to use this tool wisely. Make sure...
2. It's your responsibility to make sure...